.circleci/config.yml

```diff
@@ -824,6 +829,8 @@ jobs:
       - store_artifacts: *artifacts_test_results
 
   t_ubu_ubsan_clang_cli:
+    environment:
+      UBSAN_OPTIONS: silence_unsigned_overflow=1
```

Comment:
This seems unrelated to the PR.

Reply:
Yeah. I can remove it but felt it would be another PR just for this change. The context is that the nightly UBSan CLI job failed because UBSan printed superfluous unsigned integer overflows in boost onto stdout.

Reply:
But the PRs would be independent, wouldn't they? One would not have to wait for the other.

I really think that it's better to have it in its own PR because if I wanted to get back to it, I would not even think about looking in this PR. Or at least to adjust the PR name to mention it.

Reply:
Sounds good, I will create a new PR for it.